### ISSUE TITLE
Clarify what "offline mode" means

### ISSUE THREAD

**Issue by PureTryOut** [piggz/supersonik#3]
I see a button for it in the menu (and I saw your toot about it) but I'm unsure what "offline mode" actually means here.

My only experience with an online-music player that has offline capabilities is Spotify. In that app you can download individual songs or whole playlists at a time and if a track is available offline it is indicated as such on the entry of the track in the list.
I was expecting something like that here too but it seems to be different?

Is a track currently streamed if it's offline available but played in online mode?

**Comment by piggz** [comment]
Its currently implemented like this
In 'online mode' you will see a download button for an album, click it.
All tracks in the album are downloaded
Now switch to offline mode, and it will show your offline-available albums, and will play the local files instead of streaming them.

**Comment by PureTryOut** [comment]
Ah, thanks, that helps. Could there be an option added that if a track is available locally it is preferred over streaming it, no matter if offline mode is enabled or not?

**Comment by piggz** [comment]
Yes, I plan to add that, just needs a quick lookup of the track id.
